kubernetes-ingress-defaultbackend.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: ingress-default-backend
+  name: kubernetes-ingress-defaultbackend
   version: 1.23.1
   epoch: 0
   description: 404-server is a simple webserver that satisfies the ingress and serve pages 404 at root and 200 at /healthz
```

Comment:
nit: having this description call the software "404-server" might be confusing to some. Maybe just start with "A simple web server that..."

Reply:
done